Update Hub release note - Stale image info (#11733)
docker/docker.github.io: docker-hub/release-notes.md

Here you can learn about the latest changes, new features, bug fixes, and
known issues for each Docker Hub release.

# 2020-11-10

### New features

> * The repository views now show which images are stale (without recent push/pull activity). For more information, see [repository tags](repos.md/#viewing-repository-tags)

Reviewer: ```suggestion
* The repository views now show which images are stale (without recent push/pull activity). For more information, see [repository tags](repos.md/#viewing-repository-tags).
```

> * The repository views now show which images are stale (without recent push/pull activity). For more information, see [repository tags](repos.md/#viewing-repository-tags).

Reviewer 2: ```suggestion
* The **Repositories** view now shows which images are stale (without recent push/pull activity). For more information, see [repository tags](repos.md/#viewing-repository-tags).
```

> * The repository views now show which images are stale (without recent push/pull activity). For more information, see [repository tags](repos.md/#viewing-repository-tags).

Reviewer: ```suggestion
* The repository views now show which images have gone stale because they haven't been pulled or pushed recently. For more information, see [repository tags](repos.md/#viewing-repository-tags).
```

> * The **Repositories** view now shows which images are stale (without recent push/pull activity). For more information, see [repository tags](repos.md/#viewing-repository-tags).

Reviewer: ```suggestion
* The **Repositories** view now shows which images have gone stale because they haven't been pulled or pushed recently. . For more information, see [repository tags](repos.md/#viewing-repository-tags).
```
